README.md

```diff
@@ -57,51 +54,68 @@ Run the following commands to insert two products: Avocado and Gold
 
 ```shell
 curl -X POST localhost:8080/products/create -H 'Content-Type: application/json' \
-	 -d '{"Name":"Avocado","Price":10,"Quantity":5,"_id":"11111111-1111-1111-1111-111111111111"}'
+	 -d '{"Name":"Avocado","Price":10,"Quantity":5}'
 curl -X POST localhost:8080/products/create -H 'Content-Type: application/json' \
-	 -d '{"Name":"Gold","Price":3000,"Quantity":1,"_id":"22222222-2222-2222-2222-222222222222"}'
+	 -d '{"Name":"Gold","Price":3000,"Quantity":1}'
 ```
 
 ### Place some orders
 
+In the following requests replace the placeholders with the users and products UUIDs
+returned by the above requests, using the following rules:
+* uuid1 - UUID of John
+* uuid2 - UUID of Jane
+* puuid1 - UUID of Avocado
+* puuid2 - UUID of Gold
+* ouuid1 - UUID of successfully created order
+ 
+Check [example.sh](example.sh) for repeatable example with hardcoded UUIDs.
+
 #### Low balance
 
-The next order will fail because John is trying to purchase 1 unit of Gold which costs 3000,
-while John's balance is 100.
+Let's start off with an order that fails because John is trying to purchase 1
+unit of Gold that costs $3000.00, while John's balance is $100.00.
 
 ```shell
-curl -X POST localhost:8080/orders/create -H 'Content-Type: application/json' \
-	 -d '{"UserId":"11111111-1111-1111-1111-111111111111", "Products" : [{"_id":"22222222-2222-2222-2222-222222222222","Quantity":1}],"_id":"11111111-1111-1111-1111-111111111111"}'
+curl http://localhost:8080/orders/create \
+      -X POST \
+      -H 'Content-Type: application/json' \
+      -d '{"UserId": uuid1, "Products" : [{"id": puuid2,"Quantity":1}]}'
 ```
 
 #### Low stock
 
-The next order will fail because Jane is trying to purchase 30 Avocados which costs 300, while
-Jane's balance is 200.
+The next order fails as well because Jane is trying to purchase 30 Avocados that
```

Comment:
This example then is very similar to John's purchase above. Don't you want to show failure due to low product inventory instead?

Reply:
Actually the curl below is correct, the description is off. Fixing.